Add RabbitMQ to docker-compose + new env var for Celery Broker

mvandenburgh opened the pull request:
1) Creates a separate build stage for web server and celery worker
2) Moves the CMD block that starts the nginx `unit` server into the web server build stage, and adds a CMD block for the celery worker
3) Updates our CD workflow to build docker images for both the web server and celery worker, targeting their respective build stages.

mvandenburgh commented:
@BryonLewis these are changes needed to accommodate the AWS deployment. Will this interfere with your PRs?

BryonLewis commented:
> @BryonLewis these are changes needed to accommodate the AWS deployment. Will this interfere with your PRs?

A cursory look doesn't look like it will impact anything.  Do we want to add a rabbitMQ service to the docker-compose to make deployment as similar to local dev as possible?
Also adding the new environment variable to the template.env and docker-compose to pass it into django?

mvandenburgh commented:
I reverted the changes to the Dockerfile, as I realized we can avoid having separate images for web app + worker. This is advantageous because it will ensure they stay synced up.
 
> > @BryonLewis these are changes needed to accommodate the AWS deployment. Will this interfere with your PRs?
> 
> A cursory look doesn't look like it will impact anything. Do we want to add a rabbitMQ service to the docker-compose to make deployment as similar to local dev as possible? Also adding the new environment variable to the template.env and docker-compose to pass it into django?

I added a rabbitmq service to docker-compose, and also updated the template.env.
